exercises/collatz-conjecture/canonical-data.json

```diff
@@ -0,0 +1,48 @@
+{
+  "exercise": "collatz-conjecture",
+  "version": "1.0.0",
+  "cases": [
+    {
+      "description": "1 is 0 steps",
+      "property": "collatz",
+      "number": 1,
+      "expected": "0"
+    },
+    {
+      "description": "2 is 1 step",
```

Comment:
I'd like to see the descriptions describe what is being tested.
"first even number"